src/Form/Field/FormField.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import classNames from "classnames";
+import styled from "styled-components";
+
+const HelperMessage = styled.p`
+  margin: 0;
+
+  color: gray;
+`;
+const ErrorMessage = styled.p`
+  margin: 0;
+
+  color: red;
+`;
+
+interface FormFieldProps {
+  labelledBy: string;
+  label?: string;
+  className?: string;
+  children: any;
+  helperMessage?: string;
+  errorMessage?: string;
+}
+
+function FormField(props: FormFieldProps) {
+  const {labelledBy, label, className, children, errorMessage, helperMessage} = props;
+  const hasLabel = Boolean(label);
+  const hasErrorMessage = Boolean(errorMessage);
+  const hasHelperMessage = Boolean(helperMessage);
+  const formFieldClassName = classNames(
+    "form-field",
+    {
+      "has-error": hasErrorMessage
+    },
+    className
+  );
+
+  return (
+    <div className={formFieldClassName}>
+      {hasLabel && (
```

Comment:
label should always be placed on DOM. It's very important for assistive technologies. We can hide it using CSS if necessary.

Reply:
Yeah, but should we show even if it's empty? It's a basic existence check `hasLabel = Boolean(label)`